mmseg/models/segmentors/encoder_decoder.py

```diff
@@ -172,8 +172,8 @@ def slide_inference(self, img, img_meta, rescale):
         h_stride, w_stride = self.test_cfg.stride
         h_crop, w_crop = self.test_cfg.crop_size
         batch_size, _, h_img, w_img = img.size()
-        assert h_crop <= h_img and w_crop <= w_img, (
-            'crop size should not greater than image size')
+        # assert h_crop <= h_img and w_crop <= w_img, (
+        #     'crop size should not greater than image size')
```

Comment:
These lines could be removed. We may also add comments about what would happen if `h_crop > h_img`.